content/apidocs-mxsdk/apidocs/team-server-api.md

```diff
@@ -6,201 +6,409 @@ menu_order: 65
 
 ## 1 Introduction
 
-The team server API allows you to retrieve the information (branches, revisions) of application models stored in our team server. You always access an application model via the context of an application (see the Deploy API for more information about retrieving applications and application identifiers).
-The image below provides a domain model representation of the concepts discussed below and how these are related:
-![](attachments/131076/425989.png)
+The team server API allows you to retrieve the information (branches, commits) of application models stored in our team server. You always access an application model via the context of an application (see the Deploy API for more information about retrieving applications and application identifiers).
 
 {{% alert type="warning" %}}
 The team server API is only available to *licensed* apps which are running in a Mendix Cloud.
```

Comment:
This warning does not apply to the new API.

Reply:
Removed